db/migration/1751575915617-ChangeRecircLinksToHybridLinks.ts

```diff
@@ -0,0 +1,102 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
```

Comment:
This is a new migration that updates all instances of `"recirc-link"` to `"hybrid-link"`

Nothing bad would happen if we didn't do this, as these strings only matter for the purposes of link extraction during publishing (and would therefore naturally fix themselves if/when we republish) but just in case a dev ever needs to query the database for them, it's still good for integrity to keep everything correct.